src/main/java/htsjdk/tribble/readers/AsciiLineReader.java

```diff
@@ -36,15 +35,25 @@
  * @author jrobinso
  */
 public class AsciiLineReader implements LineReader, LocationAware {
+    private static final Log log = Log.getInstance(AsciiLineReader.class);
+
     private static final int BUFFER_OVERFLOW_INCREASE_FACTOR = 2;
     private static final byte LINEFEED = (byte) ('\n' & 0xff);
     private static final byte CARRIAGE_RETURN = (byte) ('\r' & 0xff);
 
-    private final PositionalBufferedStream is;
+    private PositionalBufferedStream is;
     private char[] lineBuffer;
     private int lineTerminatorLength = -1;
 
+    protected AsciiLineReader() {};
+
+    /**
+     * TODO: This class implements LocationAware, which requires preservation of virtual file pointers on BGZF inputs.
+     * However, if the inputStream wrapped by this class is a BlockCompressedInputStream, it violates that contract by
+     * wrapping the stream and returning positional file offsets instead.
+     */
     public AsciiLineReader(final InputStream is){
```

Comment:
Hm. We could do that. There are a bunch of internal uses of it, but thats easily fixed. But should we deprecate the other one too (the one that takes a PositionalBufferedInputStream), since "from" is the preferred method ? Otherwise its weird to leave the PBS constructor, but not have a BCS constructor (you'd have to use .from for that case). It would be most consistent to just have .from, but thats less BWC.